onnx2pytorch/operations/expand.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from torch import nn
+
+
+class Expand(nn.Module):
+    def __init__(self):
```

Comment:
The __init__ is redundant since it calls just the super. Can you remove it please?

Reply:
Done!